Add LC survey condition and links

libcce opened the pull request:
Add LC condition to add pre and post survey links to fix https://github.com/carpentries/workshop-template/issues/511

Commit:
Add LC survey condition and links

Add LC condition to add pre and post survey links to fix https://github.com/carpentries/workshop-template/issues/511

rgaiacs commented on index.md:
```diff
@@ -194,11 +194,13 @@ and our administrator may contact you if we need any extra information.</h4>
-{% elsif site.carpentry == "lc" %}
-<p>Ask your instructor about pre- and post-workshop Survey details.</p>
+{% elsif page.carpentry == "lc" %}
+<p>Please be sure to complete these surveys before and after the workshop.</p>
```
Please refactor the if-else clause to avoid duplication of the HTML. the if-else should be only for the `href`.

Commit:
Reduce repetition of HTML in survey links

Move complete survey text above if-else conditions to reduce repetition of HTML per https://github.com/carpentries/workshop-template/pull/516#discussion_r204039965 @rgaiacs
